scripts/bulk-password-change.js

```diff
@@ -0,0 +1,66 @@
+const rpn = require('request-promise-native');
+process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;
+const minimist = require('minimist');
+
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    'h': 'help'
+  }
+});
+
+if(argv.h || !argv.newPassword || !argv.url || !argv.user || !argv.password) {
+  console.log(`Set all user passwords to the provided password.
+
+Usage:
+    ${process.argv[1]} -h | --help
+    ${process.argv[1]} --newPassword Secret_1 --url https://localhost --user medic --password adminPass
```

Comment:
Maybe change this from`${process.argv[1]}` to just a hard coded `bulk-password-change.js`?  It shows a long path for me and at first I didn't see the `bulk-password-change.js` at the end so it wasn't clear this was the help text (vs an error msg).  No biggie!

```
Usage:                                                                                                                                      
    /home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js -h | --help
    /home/mrjones/Documents/MedicMobile/cht-core/scripts/bulk-password-change.js --newPassword Secret_1 --url https://localhost --user medic
 --password adminPass
```

Reply:
I'll update it